validator/accounts/v2/accounts_list_test.go

```diff
@@ -62,9 +62,11 @@ func TestListAccounts_DirectKeymanager(t *testing.T) {
 		depositData, err := wallet.ReadFileAtPath(ctx, accountName, direct.DepositTransactionFileName)
 		require.NoError(t, err)
 		depositDataForAccounts[i] = depositData
-		unixTimestamp, err := wallet.ReadFileAtPath(ctx, accountName, direct.TimestampFileName)
+		keystoreFileName, err := wallet.FileNameAtPath(ctx, accountName, direct.KeystoreFileName)
 		require.NoError(t, err)
-		accountCreationTimestamps[i] = unixTimestamp
+		timestampStart := strings.LastIndex(keystoreFileName, "-") + 1
+		timestampEnd := strings.LastIndex(keystoreFileName, ".")
```

Comment:
What if these dont exist? Will we panic below? We should have a single function that gets this timestamp from the string and fails if it cannot